CRM/Contact/BAO/GroupNestingCache.php

```diff
@@ -69,7 +69,7 @@ static public function update() {
     }
 
     if (self::checkCyclicGraph($tree)) {
-      CRM_Core_Error::fatal(ts('We detected a cycle which we cant handle. aborting'));
+      CRM_Core_Error::fatal(ts("We detected a cycle which we can't handle. aborting"));
```

Comment:
Fair enough, I had just remembered @mlutfy mentioned a while ago that low level error messages are rarely seen and hard to google if translated, so best left in English.